### ISSUE TITLE
author aspect associated images now less interesting

### ISSUE THREAD

**Issue by egonw** [fnielsen/scholia#423]
Just a note... previously, it would give a visual overview of the topics people were working on, but increasingly it's a gallery of co-author (which I'm not really interested in seeing visually...)... e.g. this is what my profile now looks like:

![image](https://user-images.githubusercontent.com/26721/42082295-e80a2db6-7b87-11e8-80f2-70d8c250dbdc.png)

**Comment by fnielsen** [comment]
But isn't it interesting for me to see your coauthors?

**Comment by Daniel-Mietchen** [comment]
I think both the people and the topics are useful to show here, and that the previous mix worked better overall.

**Comment by egonw** [comment]
@fnielsen, no, not for me... I know who my co-authors are... for other researchers I am also not interested in what their collaborators looks like. Most certainly not as much as I like to know what they do research about.
